### PR TITLE
fixing and formatting contributors list

### PR DESCRIPTION
**Pull request by burhan**
I thought I would cleanup the contributors listing, organizing the large list of contributors in alphabetical order and also highlighting separately those possessing commit flags.

### CODE REVIEW

File: CONTRIBUTORS.rst [hunk 1]
```diff
@@ -1,77 +1,143 @@
...
+  Mathijs Hoogland         `@MathijsHoogland`_
+  Matt Linares
+  Matt Warren              `@mfwarren`_
+  mozillazg                `@mozillazg`_
+  Pablo                    `@oubiga`_
+  Raphael Pierzina         `@hackebrot`_
+  Roman Afanaskin          `@siauPatrick`_
+  Russell Davies
+  stepmr                   `@stepmr`_
+  Sławek Ehlert            `@slafs`_
```
Comment (bogdal) [inline CONTRIBUTORS.rst]: The order should be reversed:
```
Sławek Ehlert
stepmr
```

**Comment by pydanny** [comment]
At first I was like, "...but the current way preserves the order of joining our growing community of contributors!" But then I was like, "Alphabetization is good, and besides, there are those pretty graphs." So this definitely has my :+1: 

If the merge conflicts are cleaned up, I'll merge this in ASAP.

**Comment by luzfcb** [comment]
Jim Munro has commit permission? (Who is Jim Munro :flushed: ?)

**Comment by pydanny** [comment]
@luzfcb In the early days he was someone I talked to about building the project. I guess he got busy with other things. In any case, I pulled his commit rights.

**Comment by luzfcb** [comment]
@pydanny thanks for explaining.

**Comment by pydanny** [comment]
Merged in!

**Pull request closed without merging**